src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -74,16 +97,29 @@ public LocalDate addWeeks(LocalDate localDate, Integer numberOfWeeks) {
      * - "someDate is today" - если someDate - сегодня
      */
     public String beforeOrAfter(LocalDate someDate) {
-        return someDate + "is today";
+        if (someDate.isAfter(LocalDate.now())) {
```

Comment:
create a variable not to ask method LocalDate.now twice

Reply:
fixed